documentation/proposals/Proposal - Multi-Backend Input.md

```diff
@@ -834,6 +831,103 @@ public interface IJoystickInputHandler : IInputHandler
 
 `HandleHatMove` must be called when any value of `JoystickState.Hats` changes.
 
+# Input Maps
+
+## Presentation
+Input maps are designed to be easyer to the user to handle actions with more than one input.
+At first view it can looks useless and a bad practice, but this is complemtelly useful for cross-plataform programs, being games an example.
+
+This is an pseudocode example of what the Input Map system solves:
+
+Without inputMaps:
+```cs
+inputContext.Update();
+
+private Dictionary<string, bool> pressedKeys = new();
+
+inputContext.Gamepads.ThumbstickMove += @event => {Move(@event.Value)};
+inputContext.Joysticks.AxisMove += @event => {Move(@event.Value)};
+
+private Move(Vector2D<float> value)
+{
+    // Code for movement here
+}
+```
+
+With inputMaps:
+```cs
+inputContext.Update();
+
+// This single line get rid of the two in the example above. TWO ISN'T THE LIMIT!
+inputContext.Map.GetAction("Move").Axis += @event => {Move(@event.Value)};
+
+private Move(Vector2D<float> value)
+{
+    // Code for movement here
+}
+```
+
+## Implementation
+Each InputContext have one reference to an InputMap.
+This is how the InputMap class work:
+```cs
+public partial class InputMap
+{
+    public InputMapAction GetAction(string name);
```

Comment:
Consider providing documentation comments and/or MoSCoW/RFC-style requirements as done throughout the 3.0 proposals. As it stands I have no idea what any of this must do and would have no idea how to implement it from this document alone.

Reply:
I really want to use more comments but i was a little afraid because I was only using this proposal as a reference. I will check how it is done in another one and try to describe better